nimble/controller/src/ble_ll_adv.c

```diff
@@ -2596,16 +2597,18 @@ ble_ll_adv_sm_start(struct ble_ll_adv_sm *advsm)
 
     /*
      * XXX: while this may not be the most efficient, schedule the first
-     * advertising event some time in the future (5 msecs). This will give
-     * time to start up any clocks or anything and also avoid a bunch of code
-     * to check if we are currently doing anything. Just makes this simple.
+     * advertising event some time in the future (5 msecs + adv rand). This will
+     * give time to start up any clocks or anything and also avoid a bunch of
+     * code to check if we are currently doing anything. Just makes this simple.
      *
      * Might also want to align this on a slot in the future.
      *
      * NOTE: adv_event_start_time gets set by the sched_adv_new
      */
+    start_delay_us = 5000 + (rand() % (BLE_LL_ADV_DELAY_MS_MAX * 1000));
```

Comment:
maybe instead `rand()` we could use `ble_ll_rand_data_get()`?

Reply:
we need random number, not random data...